Documentation/ceph-mon-health.md

```diff
@@ -17,63 +24,91 @@ To have an immutable IP address in Kubernetes, Rook creates a K8s service for ea
 When a monitor pod starts, it will bind to its podIP and it will expect communication to be via its service IP address.
 
 ## Monitor Quorum
+
 Multiple mons work together to provide redundancy by each keeping a copy of the metadata. A variation of the distributed algorithm Paxos
 is used to establish consensus about the state of the cluster. Paxos requires a super-majority of mons to be running in order to establish
 quorum and perform operations in the cluster. If the majority of mons are not running, quorum is lost and nothing can be done in the cluster.
 
 ### How many mons?
+
 Most commonly a cluster will have three mons. This would mean that one mon could go down and allow the cluster to remain healthy.
 You would still have 2/3 mons running to give you consensus in the cluster for any operation.
 
 You will always want an odd number of mons. Fifty percent of mons will not be sufficient to maintain quorum. If you had two mons and one
 of them went down, you would have 1/2 of quorum. Since that is not a super-majority, the cluster would have to wait until the second mon is up again.
+Therefore, Rook prohibits an even number of mons.
```

Comment:
You don't really prohibit, at least the operator does not, we just print a message but continue.

Reply:
Recently I actually added an enforcement because users were trying an even number and it was causing support issues. See [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/cluster.go#L320-L322).

Reply:
Oops my bad!